packages/manager/apps/web/client/app/domain/list/restore-renew/routing.js

```diff
@@ -9,7 +9,13 @@ export default /* @ngInject */ ($stateProvider) => {
     },
     resolve: {
       breadcrumb: () => null,
-      goBack: /* @ngInject */ ($state) => () => $state.go('^'),
+      goBack: /* @ngInject */ ($state, atInternet) => () => {
+        atInternet.trackClick({
+          name: 'web::domain::::pop-up::button::renew-restore-domains::cancel',
+          type: 'action',
+        });
```

Comment:
```suggestion
```
It's would be more efficient in controller file. Because if we reuse the goBack in several place the track will be sent.